CMakeLists.txt

```
@@ -436,6 +436,7 @@ message("")
 message("List of upcoming api-breaking changes:")
 message("\t Making variables non-copiable: https://github.com/KratosMultiphysics/Kratos/pull/6339")
 message("\t Deprecation of NOT_FLAGS : https://github.com/KratosMultiphysics/Kratos/pull/6233")
+message("\t Removing long deprecated elements/conditions : https://github.com/KratosMultiphysics/Kratos/pull/6475")
```

Technically only conditions
```suggestion
message("\t Removing long deprecated conditions : https://github.com/KratosMultiphysics/Kratos/pull/6475")
```